src/main/java/uk/ac/sanger/sccp/stan/model/Labware.java

```diff
@@ -185,7 +196,7 @@ public int hashCode() {
     }
 
     @JsonIgnore
-    public boolean isUsable() {
+    public boolean isStorable() {
```

Comment:
Is it not required to add 'isUsed()' here as used state is also storable?

Reply:
The method says that the labware will be storable if it is not released nor destroyed nor discarded. Used isn't any of those, so it is storable.